.devcontainer/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# [Choice] Node.js version (use -bullseye variants on local arm64/Apple Silicon): 18, 16, 14, 18-bullseye, 16-bullseye, 14-bullseye, 18-buster, 16-buster, 14-buster
+ARG VARIANT=16-bullseye
```

Comment:
Not a blocker in any sense, but you can get away without a Dockerfile for this devcontainer:
https://github.com/patheard/aws-lambda-zipper/blob/main/.devcontainer/devcontainer.json

Reply:
Terragrunt is not a feature in those I think